src/webview/MessageList.js

```diff
@@ -370,9 +370,7 @@ export default connect<SelectorProps, _, _>((state, props: OuterProps) => {
   return {
     backgroundData,
     initialScrollMessageId:
-      props.initialScrollMessageId !== undefined
-        ? props.initialScrollMessageId
-        : getFirstUnreadIdInNarrow(state, props.narrow),
+      props.initialScrollMessageId ?? getFirstUnreadIdInNarrow(state, props.narrow),
```

Comment:
I went through checking the types at each of these spots to confirm the `??` form was equivalent... and there's this one case where it isn't! The type of this property on `props` is `initialScrollMessageId?: number | null`. So if `null` is passed, this form behaves differently.

The others look good, though, and I'd be glad to have this change for the rest of them.